code/game/objects/structures/prop.dm

```diff
@@ -2177,6 +2177,52 @@
 	screen_overlay = "solar"
 	broken_icon = "computer_blue_broken"
 
+/obj/machinery/filtration_pipes
+	name = "Disinfection Filter"
+	desc = "A water filter specifically designed to separate micro-organisms, such as viruses and bacteria, from water."
+	icon = 'icons/obj/structures/pipe_multiple.dmi'
+	icon_state = "solo_tank_water"
+	density = TRUE
+
+/obj/machinery/filtration_pipes/water/update_overlays()
+	. = ..()
+	var/image/disinfection_top_overlay = new(src.icon, "tank_water_top")
+	disinfection_top_overlay.pixel_y += 31
+	disinfection_top_overlay.layer = ABOVE_ALL_MOB_LAYER
```

Comment:
you can set the layer in the new() proc (you can do pixel_y as well, I think, but not 100% sure)